Tasks/ServiceFabricDeployV1/ServiceFabricSDK/Publish-UpgradedServiceFabricApplication.ps1

```diff
@@ -359,24 +359,105 @@ function Publish-UpgradedServiceFabricApplication
             return
         }
 
-        $upgradeStatus = Wait-ServiceFabricApplicationUpgradeAction -ApplicationName $ApplicationName
+        $unhealthyEvaluationMessageFunction = {
+            param(
+                [object]$UnhealthyEvaluations,
 
-        if ($UnregisterUnusedVersions)
-        {
-            Write-Host (Get-VstsLocString -Key SFSDK_UnregisterUnusedVersions)
-            foreach ($registeredAppType in Get-ServiceFabricApplicationTypeAction -ApplicationTypeName $names.ApplicationTypeName)
+                [string]$Indentation
```

Comment:
Can Indentation have default value of "", so that it need not be passed everytime